lib/keen-query/index.js

```diff
@@ -196,7 +198,7 @@ class KeenQuery {
 
 	getAsFunnelStep () {
 		return Object.assign({}, this.query, this.buildComplexQueryParts(true), {
-			actor_property: this.query.actor_property || this.query.target_property
+			actor_property: this.query.actor_property
```

Comment:
this will mean funnels will now always need a `->with()` ... which is either 
a) a breaking change
b) fixing a 'bug'

I'm gonna go wit the latter 😅